app/models/user.rb

```diff
@@ -93,7 +93,7 @@ def self.find_for_github_oauth(auth, signed_in_resource=nil)
       user.update_attributes(params)
     else
       email =  auth.info.email
-      email =  GitHubBub::Request.fetch("/user/emails", token: token).json_body.first if email.blank?
+      email =  GitHubBub.get("/user/emails", token: token).json_body.first if email.blank?
```

Comment:
@schneems You would know, because you wrote `GitHubBub`, but I got a `NoMethodError` on `fetch`, so I checked the gem's source, and it looked like it's been replaced by `get`? Was this referencing an older version of the code?